reVX/plexos/utilities.py

```diff
@@ -99,6 +99,9 @@ class DataCleaner:
 
     REV_NAME_MAP = {'gid': 'sc_gid',
```

Comment:
You went through a great deal of effort in the PR to add a `gid_column` input to many functions that defaults to `"sc_gid"`, but it look slike you never actually use anything other than the default value of `"sc_gid"`. Do you set it to something else in your personal scripts? Or does it just work because of this mapping? If its the latter, do we actually need the `gid_column` input in all of those functions?

Reply:
ugh i dont remember! i think i was just trying to hardcode less of this stuff but i agree this seems unnecessary. Maybe will remove and hardcode?

Reply:
oh i see, note from pesap: "For the Plexos Nodal aggregation we are going to pass
the gid_column as "sc_point_gid""

https://github.com/NREL/reVX/pull/224/commits/d689c602e17fcd1b7eba53688530b14aac48f7da